### ISSUE TITLE
Unknown --project argument

### ISSUE THREAD

**Issue by Zitrax** [ArmDeveloperEcosystem/arm-learning-paths#669]
At https://learn.arm.com/learning-paths/microcontrollers/vcpkg-tool-installation/usage/ there is this info:

```
vcpkg activate --project ../my-config.json
```

But if I try that I get:

```
vcpkg activate --project .\vcpkg-configuration.json
error: unexpected switch: --project
```

And I am on a quite recent vcpkg version:

```
vcpkg --version
vcpkg package management program version 2023-12-12-1c9ec1978a6b0c2b39c9e9554a96e3e275f7556e
``` 

And that switch is not part of the help doc for vcpkg activate either.

**Comment by pareenaverma** [comment]
Hi @Zitrax thank you for reporting this issue. You are right, --project is an invalid option. You should use --json to point to the location of the json file.

The command should be:
vcpkg activate --json=../my-config.json

I will close this issue once the fix is merged into production. 

Thank you!

**Comment by Zitrax** [comment]
`--json` seem to be documented as:

> Full path to JSON file where environment variables and other properties are recorded

So that does not seem to be the way of pointing to a manifest file, actually using it overwrote my manifest file with some json output - so this is something different.

The only way I found so far that seem to do something similar is `--x-manifest-root`, although it points to a directory containing the vcpkg-configuration.json file, it does not directly point to a file.

**Comment by pareenaverma** [comment]
Hi @Zitrax thanks for reporting that back. I'm checking if there's an option to point directly to a manifest file instead of a directory containing the manifest file.

**Comment by Zitrax** [comment]
Related I found this commit: https://github.com/microsoft/vcpkg-tool/commit/3247920fbdd47d08f36cbd480addd9890d3c2435 which seem to indicate that `--project` existed earlier but was reverted for some reason.

**Comment by KeilChris** [comment]
Hello @Zitrax , you are correct - the feature has been removed. We will also remove this from the learning path. Unfortunately, the "warning: vcpkg-artifacts is experimental and may change at any time." is true...
